src/Symfony/Component/HttpKernel/EventListener/CacheAttributeListener.php

```diff
@@ -19,13 +19,14 @@
 use Symfony\Component\HttpKernel\Event\ControllerArgumentsEvent;
 use Symfony\Component\HttpKernel\Event\ResponseEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Contracts\Service\ResetInterface;
 
 /**
  * Handles HTTP cache headers configured via the Cache attribute.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class CacheAttributeListener implements EventSubscriberInterface
+class CacheAttributeListener implements EventSubscriberInterface, ResetInterface
```

Comment:
not needed either - this will make it easier to integrate:
```suggestion
class CacheAttributeListener implements EventSubscriberInterface
```

Reply:
++